src/components/hero.jsx

```diff
@@ -18,7 +18,6 @@ function Hero({ hero }) {
                 onClick={() => {
                   navigate(hero.buttonLink);
                 }}
-                className="usa-button"
```

Comment:
In my PR this will be changed to usa-button--inverse, so a class name will be used. But no change needed here

Reply:
So I can leave it as it is?

Reply:
Yes